src/utils/rfq_blockchain_utils.ts

```diff
@@ -231,4 +248,58 @@ export class RfqBlockchainUtils {
     public async isWorkerReadyAsync(workerAddress: string, balance: BigNumber, gasPrice: BigNumber): Promise<boolean> {
         return isWorkerReadyAndAbleAsync(this._web3Wrapper, workerAddress, balance, gasPrice);
     }
+
+    /**
+     * Converts a @0x/web3-wrapper `TxData` object into an Ethers `TransactionRequest`.
+     *
+     * If the `from` address is not present in the `TxData`, the function uses the address
+     * of the Ethers Wallet passed to `rfqBlockchainUtils`
+     *
+     * If the `to` address is not present, the function uses the exchange proxy address.
+     */
+    public async transformTxDataToTransactionRequestAsync(
+        txOptions: Partial<TxData>,
+        chainId?: ChainId,
+        callData?: utils.BytesLike,
+    ): Promise<providers.TransactionRequest> {
+        let from = txOptions.from;
+        if (!from) {
+            if (!this._ethersWallet) {
+                throw new Error("An Ethers wallet must be provided if no 'from' address is specified");
+            }
+            from = await this._ethersWallet.getAddress();
+        }
```

Comment:
this logic was a little unexpected (was expecting a pure fn). Thoughts on replicating this functionality by offering a `getWalletAddress` fn on `RfqBlockchainUtils`? Then a caller can stitch together this logic as needed

Reply:
Yeah Daniel didn't like it either.. maybe we just say it has to be provided

Reply:
ok done